helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -161,6 +182,22 @@ autodetect:
   prometheus: false
   istio: false
 
+################################################################################
+# Optional: batch processor config
+################################################################################
+
+# Refer https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor
+# for detailed examples
+batchConfig:
```

Comment:
I'm not sure I see the benefit for having a shared value for agent, gateway, and cluster receiver processor use. These can already be set w/ `<agent|gateway|clusterReceiver>.config.processors.batch.<field>` right?

Reply:
> These can already be set w/ <agent|gateway|clusterReceiver>.config.processors.batch.<field> right?

Yes, I had missed that. I'll remove it.